apps/repo-server/testUtils.ts

```diff
@@ -151,3 +154,11 @@ export function debugRawQuery(query: SQL<unknown>) {
 	// eslint-disable-next-line no-console
 	console.log(pgDialect.sqlToQuery(query.inlineParams()).sql);
 }
+
+export function readTestEnv() {
+	const testEnvFile = "env/test.env";
+
+	if (!existsSync(testEnvFile)) return;
+
+	process.env = parseEnvFile(readFileSync(testEnvFile, "utf-8"));
```

Comment:
You may want to change this to

`process.env = Object.assign(parseEnvFile(readFileSync(testEnvFile, "utf-8")), process.env);`

so that existing variables will not be lost and that those already defined take precedence over the values in the `.env` file.